ci/views.py

```diff
@@ -586,15 +590,36 @@ def identity(request, identity):
                         extra_tags='success'
                     )
 
+            elif 'optout_identity' in request.POST:
+                details = results.get('details', {})
+                addresses = details.get('addresses', {})
+
+                for address_type, addresses in addresses.items():
+                    for address, info in addresses.items():
+                        idApi.create_optout({
```

Comment:
The issue here is that an opt out doesn't cancel any subscriptions. So when processing subscriptions for someone who has opted out, I suspect we'll get errors.

Maybe we can look at adding an HTTP endpoint on all of the hub services, to deal with an optout from the CI? That way it's the same API across services.